go.mod

```diff
@@ -51,6 +51,11 @@ require (
 	gorm.io/gorm v1.24.5
 )
 
+require (
+	github.com/chenzhuoyu/base64x v0.0.0-20230717121745-296ad89f973d // indirect
+	golang.org/x/arch v0.5.0 // indirect
```

Comment:
Why are these two dependencies added and can they be placed in the required indirect scope below?